action.yml

```diff
@@ -2,7 +2,7 @@ name: Nightfall DLP Action
 description: Scan Pull Requests for sensitive information, like credentials & secrets, PII, credit card numbers, and more.
 runs:
   using: 'docker'
-  image: 'docker://nightfallai/nightfall_cli:v0.0.3'
+  image: 'docker://nightfallai/nightfall_code_scanner:v0.0.4'
```

Comment:
what changes are in the new image? Anything that needs to be updated in the readme or release notes?

Reply:
This version contains all the code for the circleci service, nothing related to the github action. I've created the release + notes in https://github.com/nightfallai/nightfall_code_scanner explaining that. 

Does this mean I don't need to update which image our Github action is pointing to, or should I still do so just to make sure we're pointing to the most up-to-date docker image?